packages/tokens/paragraph/src/tokens.json

```
@@ -0,0 +1,10 @@
+{
+  "nl": {
+    "paragraph": {
+      "font-size": {
+        "type": "fontSizes",
+        "value": ""
```

Er moet geen value bij zitten, dat moeten thema's zelf doen.

```suggestion
        "type": "fontSizes"
```